lms/djangoapps/program_enrollments/management/commands/tests/test_link_program_enrollments.py

```diff
@@ -4,24 +4,24 @@
 from __future__ import absolute_import
 
 from uuid import uuid4
-from testfixtures import LogCapture
 
 from django.core.management import call_command
 from django.core.management.base import CommandError
 from django.test import TestCase
-
 from edx_django_utils.cache import RequestCache
+from opaque_keys.edx.keys import CourseKey
+from testfixtures import LogCapture
+
 from lms.djangoapps.program_enrollments.management.commands.link_program_enrollments import (
-    Command,
-    INCORRECT_PARAMETER_TPL,
+    COURSE_ENROLLMENT_ERR_TPL,
     DUPLICATE_KEY_TPL,
-    NO_PROGRAM_ENROLLMENT_TPL,
+    INCORRECT_PARAMETER_TPL,
```

Comment:
Nit: while you're at, any appetite for getting rid of this `TPL` abbreviation and substituting `TEMPLATE`?  I can't help but to read `TPL` as "tuple".